assets/src/edit-story/elements/shared/getMediaProps.js

```diff
@@ -23,9 +23,18 @@
  * @param {number} focalX     X axis focal point.
  * @param {number} focalY     Y axis focal point.
  * @param {number} mediaRatio Media file ratio.
+ * @param {number} opacity    Opacity in %.
  * @return {Object} Media properties.
  */
-function getMediaProps(width, height, scale, focalX, focalY, mediaRatio) {
+function getMediaProps(
```

Comment:
Hmm. Maybe this was a case of bad naming. But `getMediaProps` was specifically aimed at the focalPoint/scale calculations. Maybe it's too much conflation for this method? On the other side, `opacity` calculations are pretty straightforward.

Reply:
Okay, will change the naming for clarity and move opacity out.